internal/servicefactory/interface.go

```diff
@@ -56,3 +56,12 @@ type ServiceFactoryGetter interface {
 	// FactoryForModel returns a ServiceFactory for the given model.
 	FactoryForModel(modelUUID string) ServiceFactory
 }
+
+// ServiceFactoryGetterFunc is a convenience type for translating a getter
```

Comment:
This appears to have come back since its relation.

Reply:
I'll fix.